.github/workflows/ci.yaml

```
@@ -0,0 +1,59 @@
+name: CI
+
+on: push
+jobs:
+  test_and_lint:
+    strategy:
+      fail-fast: false
+      matrix:
+        ci_node_total: [8]
+        ci_node_index: [0, 1, 2, 3, 4, 5, 6, 7]
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:11-alpine
+        ports:
+          - '5432:5432'
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: password
+          POSTGRES_DB: dfe-teachers-payment-service_test
+    env:
+      RAILS_ENV: test
+      DFE_TEACHERS_PAYMENT_SERVICE_DATABASE_USERNAME: postgres
+      DFE_TEACHERS_PAYMENT_SERVICE_DATABASE_PASSWORD: password
+      DFE_TEACHERS_PAYMENT_SERVICE_DATABASE_HOST: localhost
+      CI_NODE_TOTAL: ${{ matrix.ci_node_total }}
+      CI_NODE_INDEX: ${{ matrix.ci_node_index }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Setup Node
+        uses: actions/setup-node@v4
+      - name: Setup yarn
+        run: npm install -g yarn
+      - name: Install Node packages
+        run: yarn install
+      - name: Install Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.2.0
+          bundler-cache: true
+      - name: Precompile assets
+        run: bundle exec rails assets:precompile
+      - name: Linting - ShellCheck
+        run: bin/rake shellcheck
+      - name: Linting - Brakeman
+        run: bundle exec brakeman -c config/brakeman.ignore
+      - name: Linting - Standardrb
+        run: bin/rails standard
+      - name: Prepare DB
+        run: bin/rails db:schema:load
```

```suggestion
        run: bin/rails db:prepare
```
This will also load seeds.